### PR TITLE
Make the partition monitored by disk sensor configurable

### PR DESCRIPTION
**Pull request by strangechoice**
I needed a way to monitor another partition than "/", which is nearly empty on my system. Might be improvable by verifying  that the given string is a partition indeed. For the moment this should be as safe as _statvfs_ is :)

### CODE REVIEW

File: data/config [hunk 1]
```diff
@@ -22,6 +22,9 @@ ExitCommand: killall Hyprland
 # The folder, where the battery sensors reside
 BatteryFolder: /sys/class/power_supply/BAT1
 
+# The partion to monitor in disk sensor (default to "/")
+# DiskPartition: /home
```
Comment (scorpion-26) [inline data/config:26]: Can you uncomment and change it to /
I have the convention to keep defaults in ```data/config```. 
Commented out should only be used for configs, that don't have a default/default is "".

File: src/System.cpp [hunk 2]
```diff
@@ -202,10 +202,11 @@ namespace System
     DiskInfo GetDiskInfo()
     {
         struct statvfs stat;
-        int err = statvfs("/", &stat);
-        ASSERT(err == 0, "Cannot stat root!");
+        int err = statvfs(Config::Get().diskPartition.c_str(), &stat);
+        ASSERT(err == 0, "Cannot stat diskPartition!");
```
Comment (scorpion-26) [inline src/System.cpp:206]: Can you include what directory failed to stat in the error message?

**Review by scorpion-26 — COMMENTED** [review]
Thanks for the PR! 
Security wise should be fine, as statvfs can be called with any existing file or directory and still work (or error, which is caught by gBar).
I think I forgot to mention last time, but can you add the config option to module.nix too?

**Comment by strangechoice** [comment]
The requested changes are implemented now.

**Pull request merged**